### ISSUE TITLE
Version 0.18.0 is not listed

### ISSUE THREAD

**Issue by andys8** [vic/asdf-elm#6]
I wanted to try out `asdf`. Elm `0.19` is looking fine. But `0.18` is not.

* I'm surprised there is only an alpha version "0.18.0-alpha"
* The installation fails, because the URL is wrong.

```shell
asdf install elm 0.18.0-alpha 
https://dl.bintray.com/elmlang/elm-platform/0.18.0-alpha/linux-x64.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (22) The requested URL returned error: 404 Not Found
error fetching https://dl.bintray.com/elmlang/elm-platform/0.18.0-alpha/linux-x64.tar.gz
```

**Comment by andys8** [comment]
Okay, I found out that actually installing elm 0.18.0 will work fine:
```shell
asdf install elm 0.18.0
https://dl.bintray.com/elmlang/elm-platform/0.18.0/linux-x64.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 6959k  100 6959k    0     0  7287k      0 --:--:-- --:--:-- --:--:-- 7287k
```

The issue is the output of `list-all`:
```shell
asdf list-all elm
0.15.1
0.16.0
0.16
0.17.0
0.17.1
0.18.0-alpha
0.19.0
```

* Installing with other platforms than osx would still not work for `0.18.0-alpha`, because there are no packages, but probably nobody is going to do so. https://dl.bintray.com/elmlang/elm-platform/0.18.0-alpha/
* `list-all` is missing "0.18.0"

**Comment by andys8** [comment]
A bit of debugging later: It's sort function swallowing the "0.18.0" release.
https://github.com/vic/asdf-elm/blob/2b27207248074eb8013d0111e70495d8565ff6c4/bin/list-all#L30-L34

**Comment by vic** [comment]
fixed in #7
